ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -904,12 +901,6 @@ ledgerDbSwitch cfg numRollbacks trace newBlocks db =
 data LedgerDbCfg l = LedgerDbCfg {
       ledgerDbCfgSecParam :: !SecurityParam
     , ledgerDbCfg         :: !(LedgerCfg l)
-    -- ledgerDbFlushingPolicy :: FP
-    --
-    -- or
-    --
-    --
-    -- ledgerDbTryFlush  :: dbhandle -> DbChangelog l -> m ()
```

Comment:
Are we sure we won't need something like this in the future? Maybe add a TODO if you think so

Reply:
Sounds good. I'll add at TODO.